ntbs-service/Pages/LabResults/Index.cshtml

```diff
@@ -9,242 +9,240 @@
     Layout = "~/Pages/Shared/_Layout.cshtml";
 }
 
-<form-leave-checker inline-template>
-    <div>
-        <nhs-width-container container-width="Standard">
-            <h1>Lab results</h1>
-        
-            <partial name="Notifications/Edit/_SinglePageErrorSummaryPartial"/>
-        
-            @if (TempData["HasSuccessfulMatch"] as bool? == true)
-            {
-                var referenceNumber = (string) TempData["MatchedReferenceNumber"];
-                var notificationId = (int) TempData["MatchedNotificationId"];
-                <div class="flash-message flash-message--success">
-                    <h1 class="flash-message-title">
-                        @($"Reference Laboratory number {referenceNumber} has successfully been matched to")
-                        <a href="@RouteHelper.GetNotificationPath(notificationId, NotificationSubPaths.Overview)">@($" NTBS ID #{notificationId}")</a>
-                    </h1>
+<form-leave-checker inline-template><div></div></form-leave-checker>
```

Comment:
Any way to avoid the `<div></div`> mess?
Wouldn't `<form-leave-checker />` be enough?